addons/tasks/CfgTaskTypes.hpp

```diff
@@ -9,7 +9,7 @@ class CfgTaskTypes {
         displayName = "$STR_A3_cfggroups_west_blu_f_armored0";
     };
     class attack {
-        displayName = "$STR_A3_Role_Assault";
+        displayName = "$STR_A3_om_system_restui_tasktooltip";
```

Comment:
I'm not sure we can use translations from Old Man. I tried to avoid them so far.

Reply:
Should be fine since this comes from `languagemissions_f_oldman`, which CBA requires.

Reply:
Alright